e-moe/guzzle6-bundle/1.1/etc/packages/emoe_guzzle_bundle.yaml

```diff
@@ -0,0 +1,4 @@
+emoe_guzzle:
+    log:
+        enabled: true # Logging requests to Monolog
+        format: 'Guzzle: [{ts}] "{method} {uri} HTTP/{version}" {code}' # Optional log format customization
```

Comment:
hint for developers: how this option can be used. I will comment it out so default value will be used and developers will be able to change it